tests/osgi-tests/src/test/java/org/wso2/extension/siddhi/io/http/test/osgi/HttpSink.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Wrong license year

Reply:
Fixed in 4970a6d52fe797b43d60461fa05a71b8e80fd262